### PR TITLE
Minor appearence changes

### PR DESCRIPTION
**Pull request by iu2frl**
Added "images.h" with few icons for future development
Modified Lora32 V2.1 pinout as in #47 
Added WiFi informations page

### CODE REVIEW

File: src/TTGO_T-Beam_LoRa_APRS.ino [hunk 1]
```diff
@@ -54,12 +54,20 @@
...
   #define BUTTON 2                  //pin number for BUTTO
   #define BUZZER 13                 // enter your buzzer pin gpio
   const byte TXLED  = 4;            //pin number for LED on TX Tracker
+*/
+#elif LORA32_21                     // Modified as in #47
+  #define I2C_SDA 21
+  #define I2C_SCL 22
+  #define BUTTON 2                  //pin number for BUTTO
+  #define BUZZER 13                 // enter your buzzer pin gpio
+  const byte TXLED  = 4;            //pin number for LED on TX Tracker
```
Comment (iu2frl) [inline src/TTGO_T-Beam_LoRa_APRS.ino:70]: Modified as #47

File: src/TTGO_T-Beam_LoRa_APRS.ino [hunk 2]
```diff
@@ -412,15 +423,15 @@ String getSatAndBatInfo() {
   String line5;
   if(gps_state == true){
     if(InpVolts > 4){
-      line5 = "SAT: " + String(gps.satellites.value()) + "  BAT: " + String(BattVolts, 2) + "V *";
+      line5 = "SAT: " + String(gps.satellites.value()) + "  BAT: " + String(BattVolts, 1) + "V *";
     }else{
-      line5 = "SAT: " + String(gps.satellites.value()) + "  BAT: " + String(BattVolts, 2) + "V";
+      line5 = "SAT: " + String(gps.satellites.value()) + "  BAT: " + String(BattVolts, 1) + "V";
```
Comment (iu2frl) [inline src/TTGO_T-Beam_LoRa_APRS.ino:428]: Restored old 1 decimal points battery shown to allow space for BT string

File: src/TTGO_T-Beam_LoRa_APRS.ino [hunk 3]
```diff
@@ -882,6 +893,17 @@ void loop() {
...
+  // Show informations on WiFi Status
+  if (apConnected) {
+    enableOled(); // turn ON OLED temporary
+    writedisplaytext(" ((WiFi))","WiFi Client Mode","SSID: " + infoApName, "Pass: ********", "IP: " + infoApAddr, getSatAndBatInfo());
+    apConnected=false;
+  } else if (apEnabled) {
+    enableOled(); // turn ON OLED temporary
+    writedisplaytext(" ((WiFi))","WiFi AP Mode","SSID: " + infoApName, "Pass: " + infoApPass, "IP: " + infoApAddr, getSatAndBatInfo());
+    apEnabled=false;
+  }
```
Comment (iu2frl) [inline src/TTGO_T-Beam_LoRa_APRS.ino:905]: Showing WiFi informations for few seconds

Comment (mattbk) [reply src/TTGO_T-Beam_LoRa_APRS.ino:905]: Thank you for this!

**Pull request merged**